src/vapid-helper.js

```diff
@@ -90,6 +98,44 @@ function validatePrivateKey(privateKey) {
 }
 
 /**
+ * Given the number of seconds calculates
+ * the expiration in the future by adding the passed `numSeconds`
+ * with the current seconds from Unix Epoch
+ *
+ * @param {Number} numSeconds Number of seconds to be added
+ * @return {Number} Future expiration in seconds
+ */
+function getFutureExpirationTimestamp(numSeconds) {
+  const futureExp = new Date();
+  futureExp.setSeconds(futureExp.getSeconds() + numSeconds);
+  return Math.floor(futureExp.getTime() / 1000);
```

Comment:
Right, leave it as `Math.floor(futureExp.getTime() / 1000)`.

Line 110 should be fine, since you are adding seconds it doesn't matter if you are adding them in UTC or non-UTC, the number of seconds you add is going to be the same.